Formula/elasticsearch-full.rb

```diff
@@ -1,21 +1,30 @@
 class ElasticsearchFull < Formula
   desc "Distributed search & analytics engine"
   homepage "https://www.elastic.co/products/elasticsearch"
-  url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.6.0-darwin-x86_64.tar.gz?tap=elastic/homebrew-tap"
+  if OS.mac?
+    url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.6.0-darwin-x86_64.tar.gz?tap=elastic/homebrew-tap"
+    sha256 "eef8661778601a45a93824439c352116502bc92cfc751da55fec3d8459dc58d6"
+  else
+    url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.6.0-no-jdk-linux-x86_64.tar.gz?tap=elastic/homebrew-tap"
```

Comment:
Can we please not use the no-jdk distribution here, instead using the distribution that has the bundled JDK. The no-jdk distribution is not part of our long-term plans.